Client/src/Pages/Uptime/Home/index.jsx

```
@@ -58,16 +58,21 @@ const UptimeMonitors = () => {
+								color="accent"
+								onClick={() => {
+									navigate("/uptime/create");
+								}}
+								sx={{
+									fontWeight: 500,
+									whiteSpace: "nowrap",
+								}}
+							>
+								Create new
+							</Button>
+						</>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)